sled-agent/src/rack_setup/service.rs

```diff
@@ -749,7 +750,7 @@ impl ServiceInner {
                     .iter()
                     .map(|config| NexusTypes::PortConfigV2 {
                         port: config.port.clone(),
-			routes: config
+                        routes: config
```

Comment:
There are a bunch of spacing-only changes here, which `cargo fmt` added, and which `cargo fmt -- -check` now requires.  Only the `lldp` clause at 799 represents a substantive change.